Makefile

```
@@ -38,6 +39,14 @@ lint:
 	$(CARGO) clippy --all --benches --tests --examples --all-features $(TEST_FEATURES) -- -D warnings
 	$(CARGO) clippy --manifest-path $(GITHUB_TOOL_MANIFEST) --tests -- -D warnings
 
+markdownlint:
```

checkmake (0.3.2)

[warning] Target body for "markdownlint" exceeds allowed length of 5 lines (6).

(maxbodylength)